backend/src/presentation/dto/list-advised-presentations.dto.ts

```diff
@@ -0,0 +1,19 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { PresentationStatus } from '@prisma/client';
+
+export class ListAdvisedPresentationsResponse {
+  id: string;
+  submissionId: string;
+  presentationBlockId: string;
+  positionWithinBlock: number;
+
+  @ApiProperty({
+    enum: ['ToPresent', 'Presented', 'NotPresented'],
+  })
+  status: PresentationStatus;
+
+  createdAt: Date;
+  updatedAt: Date;
+  publicAverageScore?: number;
```

Comment:
Acho que em uma aula o professor mencionou que esse score só seria visualizável pela própria comissão do evento, ou seja, admins. Poderia retirar daqui desse DTO na minha opinião

Reply:
Poderíamos usar isso caso o orientador seja membro da comissão, talvez... O front poderia ter um tratamento pra só exibir de acordo com o perfil do usuário. Mas caso não considere relevante, posso remover

Reply:
Se for algo chatinho de mudar, o front pode só ignorar por enquanto também. Não é algo crítico